# Analyzing an Affinity Credit Card - Imaginary Financial Modelling Case Study

In [42]:
#Answer:  Loss rate is 3% - implies that the overall default in payment is 3% of total credit provided

Consider 1000 customers for both the scenario, divide these 1000 customers into following groups(upon your assumption and understanding about the credit lending industry)
1.customers who pay the bill within the credit-free period
2.customers paying 30 days after the last due date
3.customer paying 45 days after the last due date
4.customer paying 60 days after the last due date(remove these customers for the complete calculation of the next month,basically, you are not supposed to provide further credit services to these customers)

## Q1) Profit Percent Calculation - First Scenario 

1.customers who pay the bill within the credit-free period - 550
2.customers paying 30 days after the last due date - 250
3.customer paying 45 days after the last due date - 150
4.customer paying 60 days after the last due date - 50
ASSUMPTIONS:
i) The interest rate is simple interest.
ii) Interest is calculated on a daily basis.
iii) The average card balance is the average end of month balance.
iv) All members are more than a year old.

In [43]:
#Q1: How much profit % is generated in 1 year
OC = 25*950 #Operating costs for cards
CF = 0.065*1000*950 #Cost of funds
TOC = OC + CF  #Total operating costs

In [44]:
TOC #Excluded the 50 defaulters who will not be provided with credit services later onwards

85500.0

In [45]:
MF = 20*1000 #Membership Fee
AF = 10*1000 #affiliation fee
#First Month
IF30 = 0.15*82.1917*250 
IF45 = 0.15*123.28*150
IF60 = 0.15*164.38*50
TIF1 = IF30 + IF45 + IF60

In [46]:
#Remaining Months
TIF2 = IF30*11 + IF45*11
Revenue = MF + AF + TIF1 + TIF2

In [47]:
Revenue

101504.715

In [48]:
Profit = Revenue-TOC
ProfitPercent = (Profit/Revenue)*100
ProfitPercent

15.76745966923802

## Q1) Profit Percent Calculation - Second Scenario

i) The interest rate is compounded quarterly.
ii) Interest is calculated on a monthly basis.
iii) Membership grows at 5% month on month.
iv) Card balance grows at 5% per month for each member.
v) The average card balance is the average end of month balance.

In [52]:
def compound_interest(principle, rate, time): 
  
    # Calculates compound interest  
    Amount = principle * (pow((1 + rate / 100), time)) 
    print("Amount is", Amount) 
  
compound_interest(1000,5,12)

Amount is 1795.8563260221301


Therefore, there are 1796 members at the end of the year and average card balance is 1795.8563 at the end of last month

In [61]:
AvgBal = 1000
r= range(0,13)
for i in r:
    TCF= 0
    CF = AvgBal*0.065 
    TCF = TCF + CF
    AvgBal= AvgBal*1.05
    i=i+1
    
TCF
   

116.73066119143846

In [63]:
OC= 25*1000
COST = TCF*950 + OC  #Multiplying Total cost of funds by 950 as the 50 people wo fail to repay within 60 days are excluded from further calculations
COST

135894.1281318665

In [64]:
AvgBal = 1000
r= range(0,13)
for i in r:
    IF30 = 0
    IF1 = AvgBal*0.15*250*2.5 
    IF30 = IF30 + IF1
    IF45 = 0
    IF2 = AvgBal*0.15*3.75
    IF45 = IF45 + IF2
    AvgBal= AvgBal*1.05
    i=i+1
IntFund = IF30+ IF45 + 50*0.15*1000 #last one is the interest fund generated by the first month revolvers who failed to pay in the 60 day period
IntFund

176871.69974796212

In [65]:
MF = 20*1000 #Membership Fee
AF = 10*1000 #affiliation fee

In [75]:
Revenue2= MF + AF + IntFund
Revenue2

206871.69974796212

In [67]:
Profit2 = Revenue2 - COST 
Profit2

70977.5716160956

In [69]:
#Percentage is
Perc = (Profit2/Revenue2) * 100
Perc

34.30994751943822

## Q2) The 3% loss rate can either mean either 3% of the outstanding balance defaults or 3% of the borrowers default on their entire balance. If the average balance is increased to $2000, what happens to the loss? 

Answer: If average balance is increased to $2000 from $1000, the loss will increase from $30 to $60.

## Q3) Is a borrower with a low balance more or less likely to default then a borrower with a high balance? Why ? explain clearly

Answer : A borrower with lower balance is more likely to default as - If credit card borrowers had to bear a larger
share of their debt, this might provide a sufficient incentive to more closely align credit card borrowing with the ability to repay the debts. It is usually the borrowers with lower balance who would result in defaulting of payments as they are aware of their inability to repay large sums.

## Q4) The group company has now become unhappy with the credit card provider. it wants to start its own credit card. But for that, it will need to purchase the customers from the current credit card company. If the affilated group offered Rs 20 per account to purchase all the customers from the credit card company, what would their profit be? (assume that we are only talking about 1 year)

In [72]:
#First Scenario 
TotalCost = 85500 #Taken from above calculations 
PlusAcquisitionCost = 20*1000
FinalCost = TotalCost + PlusAcquisitionCost
FinalCost

105500

In [73]:
Revenue = 101504.715
Profit = Revenue-FinalCost
ProfitPercent = (Profit/Revenue)*100
ProfitPercent

-3.9360585367881713

In [74]:
#Second Scenario
TotalCost = 135894.1281318665
PlusAcquisitionCost = 20*1796 
FinalCost = TotalCost + PlusAcquisitionCost 
FinalCost

171814.1281318665

In [76]:
Revenue = 206871.69974796212
Profit = Revenue-FinalCost
ProfitPercent = (Profit/Revenue)*100
ProfitPercent

16.946528528941986

## Q5) What numbers can the financial institution change to convince the affiliated group not to want to purchase, while still not going into a loss? Give a range of what the new numbers are (min and max)

Answer : They can reduce the affiliation fee per card to zero while still making profits. Further,  membership fee per card or interest rate can be reduced too such that no losses are still being incurred.

## Q6) What is the extra cost(if any) company will have to incur if the customer delays the payment of the credit card bill after the due date?

The company has to incur the 6.5% cost of funds if customer delays the payment of credit card bill although this cost is usually recovered through late payment fee that is charged by the credit card company

## Q7) Is it beneficial for the company if the customer pays the credit card bill after the due date? (express your views as a business strategy analyst with a short example)

Answer : If a customer is re-paying after the due date consistently, then it is a matter of concern as there is a high probability of such a customer defaulting their payments and causing losses. However, it is possible that once in a while a customer may miss their payment date and they can be charged with late fee and the costs can then be hence recovered. 

Thus, the answer to this question largely depends on the profile of each specific customer and each case.

## Q8) Which information variables you will use to calculate the credit card limit and the risk associated with it for an individual customer? Also, give the reason for your choice.

1) Disposable Income of the concerned person 
   Credit limit largely depends on the income of the concerned person as they should not be spending too much as compared to their capacity to repay.
2) Credit Bureau Score
   Clearly, an individual's credit score largely indicates their ability to repay debt as it indicates their ability to repay debt. One with low credit score shall not be given a very high limit as the money may not be recovered. 
3) Assets/Net-Worth of the Individual
   The liquid as well as illiquid assets of the individual should be taken into account as they can be used to repay debt and also measure the financial position of an individual.

   